addons/mail/static/src/core/public_web/messaging_menu.xml

```diff
@@ -10,6 +10,29 @@
         <t name="before-content"/>
         <DiscussContent t-if="['inbox', 'starred'].includes(store.discuss.activeTab) and ui.isSmall"/>
         <div t-else="" class="o-mail-MessagingMenu-list d-flex flex-column overflow-auto o-scrollbar-thin flex-grow-1 list-group list-group-flush o-scrollbar-thin" t-ref="notification-list">
+            <t t-set="itemIndex" t-value="0"/>
+            <t t-foreach="visibleStandaloneMessages" name="standaloneInboxMessages" t-as="message" t-key="message.id">
+                <NotificationItem
+                    isActive="state.activeIndex === itemIndex"
+                    counter="1"
```

Comment:
Can remove `counter`: it shows a red pill which is more than enough.
Should also have been in 19.0.